tests/lib/rules/no-hooks-from-ancestor-modules.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * @fileoverview disallow the use of hooks from ancestor modules
+ * @author Raymond Cohen
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/no-hooks-from-ancestor-modules"),
+    RuleTester = require("eslint").RuleTester;
+
+//------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+function createError({ invokedMethodName, expectedHookIdentifierName, usedHookIdentifierName, containingModuleDescription }) {
```

Comment:
You can also add some plain unit tests for this function at the bottom of this file.

Reply:
For what it's worth, I don't think it's necessary to test a one-statement helper function in the test when it's pretty clear what it should do.

Reply:
Oh true. My goal was to ensure the final error string looks right. But now I realized the string is not actually exposed here.

Reply:
I'll clean up the things I ended up not using